### PR TITLE
Add nginx roles to plays and add new plays.

### PR DESCRIPTION
**Pull request by feanil**
Added new plays for splunkforwarder and datadog and added
nginx to the xqueue, xserver, and edxapp plays.

### CODE REVIEW

File: playbooks/edx-east/edxapp.yml [hunk 1]
```diff
@@ -5,4 +5,9 @@
   vars_files:
     - ["{{ secure_vars }}", "dummy.yml"]
   roles:
+    - role: nginx
+      nginx_sites:
+        - lms
+        - cms
+        - lms-preview
```
Comment (jarv) [inline playbooks/edx-east/edxapp.yml]: don't need lms-preview

**Comment by jarv** [comment]
:+1:  should probably do the others too, certs, ora, discern

**Comment by feanil** [comment]
@jarv  Updated but I didn't see an nginx template for the certs role.

**Comment by jarv** [comment]
https://github.com/edx/configuration/blob/master/playbooks/edx-east/certs.yml
:+1:

**Pull request merged**